README.md

```diff
@@ -292,3 +292,4 @@ Questions can be asked by raising an `Issue`.
 - [Vladyslav Kyiashko](https://github.com/vlakyi)
 - [Yashasvi Misra](https://github.com/yashasvimisra2798)
 - [Yaswanth Modepalli](https://github.com/Yaswanth14)
+- [Mayur Gupta](https://github.com/Mayurmg)
```

Comment:
Lgtm-!

Reply:
ok sure

Reply:
Really sorry but i dont know why i am unable to do so. Please pass it like this only. If you can.

Reply:
Is there anything you need help with?